src/components/results/securityanalysis/security-analysis-table.tsx

```diff
@@ -56,13 +73,32 @@ export const SecurityAnalysisTable: FunctionComponent<
     );
 
     return (
-        <CustomAGGrid
-            rowData={rowsToShow}
-            columnDefs={columnDefs}
-            defaultColDef={defaultColDef}
-            onGridReady={onGridReady}
-            overlayNoRowsTemplate={overlayNoRowsTemplate}
-            {...agGridProps}
-        />
+        <Box sx={styles.gridContainer}>
+            <Box sx={styles.csvExport}>
+                <Box style={{ flexGrow: 1 }}></Box>
+                <Box>
+                    <FormattedMessage id="MuiVirtualizedTable/exportCSV" />
+                </Box>
+                <Box>
+                    <IconButton
+                        disabled={!rowsToShow || rowsToShow.length === 0}
+                        aria-label="exportCSVButton"
+                        onClick={exportCsv}
+                    >
+                        <GetAppIcon />
+                    </IconButton>
+                </Box>
```

Comment:
Wouldn't it be interesting to reuse and evolve `CsvExport` ?

Reply:
done for the visual aspect
They're not doing the same thing technically speaking though, one is using AGGrid to create CSV from frontend, the other is fully fetching the csv from the back